src/components/UserInfo.js

```diff
@@ -1,18 +1,23 @@
 export default class UserInfo {
-  constructor(nameSelector, jobSelector) {
+  constructor(nameSelector, jobSelector, avatarSelector) {
     this._nameElem = document.querySelector(nameSelector);
     this._jobElem = document.querySelector(jobSelector);
+    this._avatarElem = document.querySelector(avatarSelector);
   }
 
   getUserInfo() {
     return {
       name: this._nameElem.textContent,
       job: this._jobElem.textContent,
+      avatar: this._avatarElem.src,
     };
   }
 
-  setUserInfo(name, job) {
+  setUserInfo({name, job, avatar}) {
```

Comment:
Needs correction: It's needed to create a separate setter for avatar `setAvatar` and use it when needed to update the avatar, because user info and avatar are controlled by different forms.

When initializing user info you can use bit methods:

```js
userInfo.setUserInfo({ title, description })
userInfo.setAvatar(avatar)
```